ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-templates.yml

```diff
@@ -47,6 +47,8 @@ default_templates:
     interface_descriptions:
       underlay_ethernet_interfaces: 'interface_descriptions/underlay/ethernet-interfaces.j2'
       underlay_port_channel_interfaces: 'interface_descriptions/underlay/port-channel-interfaces.j2'
+      underlay_ethernet_mlag_interfaces: 'interface_descriptions/underlay/ethernet-mlag-interfaces.j2'
```

Comment:
IMO it would be simpler to just use `mlag_ethernet_interfaces` and place in `interface_descriptions/mlag/ethernet-interfaces.j2`. If you don't agree, at least consider renaming to `underlay_mlag_ethernet_interfaces` (same for port-channels). Thanks